Alpha fails to build because ioctl_req_t is defined as unsigned long int.

$OpenBSD: patch-src_baconfig_h,v 1.2 2010/07/20 14:38:39 merdely Exp $
--- src/baconfig.h.orig	Tue Apr 27 15:58:29 2010
+++ src/baconfig.h	Thu Jun 24 09:37:47 2010
@@ -727,6 +727,11 @@ extern "C" int setdomainname(char *name, int namelen);
 #endif /* HAVE_HPUX_OS */
 
 
+#if defined (__OpenBSD__) && defined (__alpha__)
+#undef  ioctl_req_t
+#define ioctl_req_t unsigned long int
+#endif
+
 #ifdef HAVE_OSF1_OS
 extern "C" int fchdir(int filedes);
 extern "C" long gethostid(void);
